packages/server/src/integrations/mysql.ts

```diff
@@ -242,7 +242,7 @@ module MySQLModule {
       const input = this._query(json, { disableReturning: true })
       let row
       // need to manage returning, a feature mySQL can't do
-      if (operation === "awdawd") {
```

Comment:
I chuckled at this lol

Reply:
yeap that must have been missed at some point haha